package.json

```diff
@@ -54,7 +55,6 @@
     "node": ">= 6.9.0 || >= 8.9.0"
   },
   "peerDependencies": {
-    "node-sass": "^4.0.0",
```

Comment:
Why did you remove `node-sass` as a peerDependency when `sass-loader` explicitly requires a version?

Reply:
`sass-loader` requires *either* `node-sass` or `sass`.  There's no way to express that as a peer dependency without having npm complain about one or the other missing.

Reply:
Ah! Thank you. Didn't know you could use either.